unattended_scripts/wazuh_install.sh

```diff
@@ -211,44 +272,66 @@ main() {
     importFunction "wazuh-cert-tool.sh"
 
     checkArch
+
+    
     
     if [ -n "${certificates}" ] || [ -n "${AIO}" ]; then
         createCertificates
     fi
 
+    if [ -n "${elasticsearch}" ] && [ -n "${kibana}" ]; then
+        mult_install=1
+    fi
+    if [ -n "${elasticsearch}" ] && [ -n "${wazuh}" ]; then
+        mult_install=1
+    fi
+    if [ -n "${kibana}" ] && [ -n "${wazuh}" ]; then
+        mult_install=1
+    fi
+
     if [ -n "${elasticsearch}" ]; then
 
         importFunction "elasticsearch.sh"
 
+
+        progressbar_total=8
```

Comment:
Initialize this with production total stages to 7, if -a flag is specified, change it to 12, and if development flag is specified, change value to 8 or 13, or the relevant values.

This will avoid to run ((progressbar_status--))

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/1080/commits/b6529a2b3b0eeb287bd573b77284f34b2fef99a6